Resources/Prototypes/_ADT/Entities/Clothing/Hands/rings.yml

```
@@ -12,6 +12,9 @@
     size: Tiny
   - type: StaticPrice
     price: 1000
+  - type: Tag # DS14
```

```suggestion
  - type: Tag
```
Файл не визардский поэтому тут не нужен комментарий DS14